app/src/modules/settings/routes/data-model/fields/components/field-select.vue

```diff
@@ -219,6 +219,7 @@ export default defineComponent({
 		const showRelatedCollectionLink = computed(
 			() =>
 				relatedCollectionInfo.value !== null &&
+				props.field.collection !== relatedCollectionInfo.value.relatedCollection &&
```

Comment:
Lets use `unref`'s here instead of `x.value`

Reply:
Marked it as a comment instead of a "changes requested" as it's a tiny nitpick that we only recently started using ourselves 🙂

Reply:
But the performance! That's one unnecessary ternary! 😄